portal/models.py

```diff
@@ -106,15 +113,14 @@ class Host(models.Model):
 
     notes = models.TextField(blank=True)
 
-    def __unicode__(self):
+    def __str__(self):
         return self.name
 
     def fqdn(self):
-        return "%s.hamwan.net" % self.name
+        return f'{self.name}.{ROOT_DOMAIN}'
```

Comment:
quick note so I don't forget:

because ROOT_DOMAIN is set to "hamwan.org" in settings.py, this change will change the fqdn for all hosts in production (unless productio overrides ROOT_DOMAIN to be "hamwan.net")

Is that the desired change? This also breaks tests

Reply:
The intent of this change was to make the portal code organization independent.  We should update the tests to accommodate this.  I basically tried to move any organization specific information to settings.py. I am already using this code for another application.